packages/design-system/src/components/Button/button.example.html

```diff
@@ -75,6 +75,53 @@ <h6 class="preview__label">Transparent button</h6>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--focus">Focus</button>
 <button type="button" class="ds-c-button ds-c-button--transparent" disabled>Disabled</button>
 
+<h6 class="preview__label">Default button inverse</h6>
+<div class="example--inverse">
+  <button type="button" class="ds-c-button ds-c-button--inverse">
+    Default
+  </button>
+  <button type="button" class="ds-c-button ds-c-button--hover ds-c-button--inverse">
+    Hover
+  </button>
+  <button type="button" class="ds-c-button ds-c-button--active ds-c-button--inverse">
+    Active
+  </button>
+  <button type="button" class="ds-c-button ds-c-button--focus ds-c-button--inverse">
+    Focus
+  </button>
+  <button type="button" class="ds-c-button ds-c-button--inverse" disabled>
+    Disabled
+  </button>
+</div>
+
+<h6 class="preview__label">Transparent button inverse</h6>
```

Comment:
Could we combine these inverse buttons to the use same `example--inverse` container? I know it gets a bit tricky because there are 2 sets of inverse buttons with 2 labels, but this is the only component example with multiple inverse sections.

Maybe we could have 1 inverse container and 1 label thats just "Inverse buttons"? I know its not as descriptive but I think it's acceptable as the label would be repetitive anyway. @line47 thoughts?
<img width="562" alt="Screen Shot 2020-04-29 at 10 49 18 AM" src="https://user-images.githubusercontent.com/5424713/80616922-1631fb80-8a07-11ea-8c62-8b2f9fe88330.png">